scripts/run_e2e_env.sh

```diff
@@ -6,7 +6,7 @@ function start_app() {
   nohup node --loader ts-node/esm packages/cli/test/scripts/e2e_test_env.ts > test-logs/e2e-test-env/simulation.out 2>&1 &
   echo $! > test-logs/e2e-test-env/simulation.pid
   echo "Wait for the node to be ready"
-  npx wait-port -t 60000 0.0.0.0:5001
+  npx wait-port -t 90000 0.0.0.0:5001
```

Comment:
Saw frequent timeout on `e2e` tests.

Reply:
Can we have a separate PR for this please, I think we need transparency on what changes we do to e2e / sim tests to have better transparency on what worked and what didn't as we did changes in the past to those tests that actually made them worse.

Reply:
It's more important to merge a PR with CI passing than tracking changes to one line. Tracking can always be done no matter how code is changed.

Reply:
Then we can open a PR that increases the timeout and merge unstable into this PR. Unless the change in this PR requires a change to e2e tests or their setup it should not be part of it.

Reply:
Would be good to understand as well why we need to increase the timeout all of the sudden, the other PR ideally includes some context around this.